core/controllers/admin.py

```diff
@@ -632,3 +633,16 @@ def get(self):
             'data': extracted_answers
         }
         self.render_json(response)
+
+
+class SendDummyMailToAdminHandler(base.BaseHandler):
+    """This function handles sending test emails."""
+
+    @acl_decorators.can_access_admin_page
+    def post(self):
+        username = self.username
+        if feconf.CAN_SEND_EMAILS:
+            email_manager.send_dummy_mail_to_admin(username)
+            self.render_json({'msg': 'Success! Mail sent to admin.'})
```

Comment:
> You don't need all this stuff. Just do self.render_json({}) at the end. (In general, follow the example of the surrounding code in the file.)

In a previous comment (copied above), I explained that you don't need to send this message to the frontend. You marked that comment as "Done", but the message is still here.

Reply:
I thought it was necessary for testing but I have removed it now.